src/test/java/org/minions/devfund/mauricioz/linkedList/InsertSortedTest.java

```diff
@@ -0,0 +1,144 @@
+package org.minions.devfund.mauricioz.linkedList;
+
+import java.util.LinkedList;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+// Source code recreated from a .class file by IntelliJ IDEA
+// (powered by Fernflower decompiler)
+*/
+public class InsertSortedTest {
+    public InsertSortedTest() {
+    }
+
+    /**
+     * setup for test.
+     */
+    @Before
+    public void setUp() throws Exception {
+    }
+
+    /**
+     * test.
+     */
+    @Test
+    public void testInsertMiddle() {
+        LinkedList var1 = new LinkedList();
+        var1.add(3);
+        var1.addLast(8);
+        var1.addLast(11);
+
+        try {
+            LinkedListUtils.insertSorted(var1, 10);
+        } catch (Exception var3) {
+            Assert.fail("insertSorted throws " + var3 + " when placing value in middle of list");
+        }
+
+        Assert.assertTrue("insertSorted creates LinkedList of incorrect size when placing value in middle of list", var1.size() == 4);
+        Assert.assertTrue("insertSorted does not correctly insert element when placing value in middle of list", (Integer)var1.get(0) == 3);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define a constant instead of duplicating this literal "insertSorted does not correctly insert element when placing value in middle of list" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)